apps/admin-gui/src/app/users/pages/user-detail-page/user-assignments/user-assignments.component.html

```diff
@@ -0,0 +1,62 @@
+<h1 class="page-title">{{'USER_DETAIL.ASSIGNMENTS.TITLE' | translate}}</h1>
+<mat-spinner class="ms-auto me-auto" *ngIf="initLoading"></mat-spinner>
+<div class="d-flex">
+  <perun-web-apps-facility-search-select
+    *ngIf="!(initLoading || facilities.length === 0)"
+    class="w-50"
+    [facilities]="facilities"
+    (facilitySelected)="loadFacility($event)">
+  </perun-web-apps-facility-search-select>
+</div>
+<mat-tab-group *ngIf="!(initLoading || facilities.length === 0)" mat-stretch-tabs="false">
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'USER_DETAIL.ASSIGNMENTS.OVERVIEW' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <mat-spinner class="ms-auto me-auto" *ngIf="loading"></mat-spinner>
+      <p class="mt-4 subtitle">{{'USER_DETAIL.ASSIGNMENTS.RESOURCES' | translate}}</p>
+      <perun-web-apps-resources-list
+        *ngIf="!loading"
+        [resources]="resources"
+        [displayedColumns]="['id', 'name', 'vo', 'description']"
+        (refreshTable)="loadFacility(selectedFacility)">
```

Comment:
Not valid for this use case.
```suggestion
      <perun-web-apps-resources-list
        *ngIf="!loading"
        [resources]="resources"
        [displayedColumns]="['id', 'name', 'vo', 'description']">
```